src/__tests__/components/profile/Profile.test.js

```diff
@@ -21,7 +22,7 @@ describe("profile component tests", () => {
   });
 
   it("should render when mounted", () => {
-    const wrapper = mount(<Provider store={store}><Profile /></Provider>);
+    const wrapper = mount(<Provider store={store}><Router><Profile /></Router></Provider>);
```

Comment:
why did you add `Router` on the profile component

Reply:
The Router is added because in our Profile component we are using a Link prop and this prop must be wrapped in a Router,  I do that in the tests and that way the Component is able to mount